cmd/forwarder/run/run.go

```diff
@@ -256,6 +260,22 @@ func (c *command) runE(cmd *cobra.Command, _ []string) (cmdErr error) {
 	return g.Run()
 }
 
+func (c *command) registerErrorsMetric() (func(name, format string), error) {
+	m := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: c.httpProxyConfig.PromNamespace,
+		Name:      "errors_total",
+		Help:      "Number of errors",
+	}, []string{"name"})
+
+	if err := c.promReg.Register(m); err != nil {
+		return nil, err
+	}
+
+	return func(name, format string) {
```

Comment:
nit: `format` is not used -> `_`

Reply:
Don't we have a linter for that? 🤔

Reply:
It's for the future, probably not worth it.